plugins/obs-webrtc/data/locale/en-US.ini

```diff
@@ -0,0 +1,4 @@
+Module.Description="OBS WebRTC module"
```

Comment:
this can be removed because it's not localized in other outputs; i don't think it's user facing anyway.